arm/Microsoft.SignalRService/webPubSub/.test/deploy.full.test.bicep

```diff
@@ -0,0 +1,22 @@
+var location = 'westeurope'
```

Comment:
We do not yet support `bicep` test files. Please provide a regular JSON-Parameter file until the CI environment supports it